README.md

```diff
@@ -1,10 +1,16 @@
-**Preview Release**
+# AngularDart Components 
```

Comment:
Don't add this header. It's redundant it the places we care about: GitHub and pub site